model/src/main/java/org/projectnessie/error/ErrorCode.java

```diff
@@ -34,7 +34,9 @@
   BAD_REQUEST(400, NessieBadRequestException::new),
   FORBIDDEN(403, NessieForbiddenException::new),
   TOO_MANY_REQUESTS(429, NessieBackendThrottledException::new),
-  ;
```

Comment:
The idea of having the `;` on a new line was to avoid changing old lines when new enum values are added :)